### DISCUSSION TITLE
ScriptReference/PrefabUtility.SaveAsPrefabAsset

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#336] [Notes]
# ScriptReference/PrefabUtility.SaveAsPrefabAsset

https://docs.unity3d.com/ScriptReference/PrefabUtility.SaveAsPrefabAsset.html

**Comment by houkanshan** [comment]
Example of the assetPath: `Assets/Prefabs/PrefabA.prefab`
